tests/modules/asset_groups/resources/test_create_asset_group.py

```diff
@@ -254,22 +277,52 @@ def test_create_asset_group_detection(
         data.add_filename(0, test_filename)
         data.set_field('speciesDetectionModel', ['someSortOfModel'])
 
-        resp_msg = 'failed to start detection'
-        asset_group_utils.create_asset_group(
-            flask_app_client, None, data.get(), 400, resp_msg
-        )
-        # TODO this is what the response should be
-        # resp_msg = 'Invalid submitter data'
-        # asset_group_utils.create_asset_group(
-        #    flask_app_client, None, data.get(), 403, resp_msg
-        # )
-
+        with mock.patch.object(
+            flask_app.acm,
+            'request_passthrough_result',
+            side_effect=detection_started,
+        ):
+            resp = asset_group_utils.create_asset_group(
+                flask_app_client, None, data.get()
+            )
+            asset_group_uuid = resp.json['guid']
+
+        from invoke import MockContext
```

Comment:
This test should be in `tests/tasks/app/test_job_control.py`

Reply:
Fair comment, this had become two tests for the price of one :)